src/BoxAnnotations.ts

```diff
@@ -1,10 +1,16 @@
 import getProp from 'lodash/get';
+import { IntlShape } from 'react-intl';
 import DocumentAnnotator from './document/DocumentAnnotator';
 import { Permissions, PERMISSIONS, Type } from './@types';
 
+type AnnotatorOptions = {
+    intl?: IntlShape;
```

Comment:
Can we merge this with the `intl` option that is already [passed in to the Annotator](https://github.com/box/box-annotations/blob/master/src/common/BaseAnnotator.ts#L32)? It's a little confusing to have two options with the same name.

Reply:
Yes... I will open a PR in the Preview SDK that will pass this option to the annotations instance and it will get forwarded to the annotator as an option.